src/rp2/plugin/country/es.py

```diff
@@ -0,0 +1,53 @@
+# Copyright 2023 Larision
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+
+from typing import Set
+
+from rp2.abstract_country import AbstractCountry
+from rp2.rp2_main import rp2_main
+
+
+# ES-specific class
+class ES(AbstractCountry):
+    def __init__(self) -> None:
+        super().__init__("es", "eur")
+
+    # Measured in days
+    def get_long_term_capital_gain_period(self) -> int:
+        return 365
+
+    # Default accounting method to use if the user doesn't specify one on the command line
+    def get_default_accounting_method(self) -> str:
+        return "fifo"
+
+    # Set of accounting methods accepted in the country
+    def get_accounting_methods(self) -> Set[str]:
+        return {"fifo"}
+
+    # Default set of generators to use if the user doesn't specify them on the command line
+    def get_report_generators(self) -> Set[str]:
+        return {
+            "es.open_positions",
```

Comment:
I don't think you don't need a Spain-specific version of open_positions (see the [Japanese plugin](https://github.com/eprbell/rp2/blob/main/src/rp2/plugin/country/jp.py#L45) as an example):
```suggestion
            "open_positions",
```

This means you can also delete `src/rp2/plugin/report/es/open_positions.py` from this PR.

Reply:
This is still unmodified (I think this is why you're still seeing the wrong currency). You need to:
* remove `es.` and 
* delete src/rp2/plugin/report/es/open_positions.py